Providers/Profile.swift

```diff
@@ -502,21 +486,6 @@ public class BrowserProfile: Profile {
         return SQLiteLogins(db: self.loginsDB)
     }()
 
-    // This is currently only used within the dispatch_once block in loginsDB, so we don't
-    // have to worry about races giving us two keys. But if this were ever to be used
-    // elsewhere, it'd be unsafe, so we wrap this in a dispatch_once, too.
-    lazy private var loginsKey: String? = {
-        let key = "sqlcipher.key.logins.db"
```

Comment:
Turns out the migration completely removed the code that grabs the key from the keychain. This just looks like it returns a basic string heh. I've added back in the keychain code and made it static.